src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -176,15 +192,23 @@ public static Tag parseTag(String tag) throws ParseException {
     }
 
     /**
-     * Parses {@code Collection<String> names} into a {@code UniquePersonList}.
+     * Parses {@code Collection<String> names} into a {@code UniquePersonList}
+     * that only contains the names of Persons to be filtered for later.
+     * Uses a blank string for UUID to enhance testability, since the UUID
+     * from here does not actually get used anywhere.
      */
     public static UniquePersonList parseContacts(Collection<String> contacts)
             throws ParseException {
         requireNonNull(contacts);
         final UniquePersonList personList = new UniquePersonList();
         for (String name : contacts) {
             Person person = new Person(
-                    parseName(name), null, null, null, new HashSet<>(), UUID.randomUUID()
+                    parseName(name),
+                    Phone.EMPTY_PHONE,
+                    Email.EMPTY_EMAIL,
+                    Address.EMPTY_ADDRESS,
+                    new HashSet<>(),
+                    UUID.fromString("e26616c9-c740-4d86-861e-733a4d377a3e")
```

Comment:
The UUID here just uses the one from Alice Pauline in the typical persons addressbook. 
It's just a placeholder that doesn't actually get used anywhere.